src/util/gentle-register-property.js

```diff
@@ -31,7 +31,7 @@ export default function gentleRegisterProperty (property, meta = {}, window = gl
 
 	if (
 		!property.startsWith("--") ||
-		!CSS.registerProperty ||
+		!CSS?.registerProperty ||
```

Comment:
This will fail if `CSS` is not defined, it will only make a difference if `CSS` is defined, but `null` or `undefined`. Is that the case in Next.js? If it's not defined at all, you'd need `globalThis.CSS?.`

Reply:
`CSS` is defined 4 lines above, though. 😁

Reply:
But yes, `window.CSS` (where `window` is actually `globalThis`, see the default value on line 29) is `undefined` in Node.

Reply:
Updated [this](https://github.com/LeaVerou/style-observer/pull/109/commits/22a0ba4f72d4dc0345a105b9aa2262ca8c061dd7#diff-d5848aa53f74b935d8cb04711a71b56953f88262a5d26db65d7388f170ebf694L30) and [this](https://github.com/LeaVerou/style-observer/pull/109/commits/22a0ba4f72d4dc0345a105b9aa2262ca8c061dd7#diff-1a79af5a5a88de3c12a6aea6bbaf68e89a0fbfff6b8f3406a4875ccb3aab3f08L12) to make them consistent with one another. Let me know if you prefer something else!